docs/_source/docs/permissions.rst

```diff
@@ -86,6 +86,18 @@ As a resolvable property, Sceptre allows you to use a resolver to populate the `
 Stack or StackGroup Config. This means you could define that role within your project, output its
 ARN, and then reference it using `!stack_output`.
 
+iam_role_session_duration
```

Comment:
I don't we need this parameter documented on this page. This page is all around designing Sceptre permissions, but this parameter is really just a detail. The Stack Config docs is probably the only place we need to document it.